app/services/db/identity/sp_active_user_counts.rb

```diff
@@ -8,7 +8,8 @@ def self.call(start)
         }
         sql = format(<<~SQL, params)
           SELECT
-            issuer,
+            service_providers.issuer,
+            MAX(app_id) AS app_id,
```

Comment:
are app ids always increasing?

Reply:
The query is definitely misleading.  app_id is not always increasing and it's an alphanumeric.  Since it's part of an aggregation it also needs to be in the `group by` or have an aggregate function on it.  So I threw `max` on it to select just one.  The app_id for a given issuer will always be the same but in the join you are getting multiple rows back and need to select one.  Putting it in the group by doesn't seem right especially because it can be null.

Reply:
Ok yea so it was to get rid of NULLs -- that was my other guess. This makes sense

Reply:
The nulls will be lumped together in the group by so it would work by putting it there.  It just felt weird.  Let me know if you think it's clearer.  Then no max(app_id) anywhere and it's just listed in the `group by`